ipa-core/src/bin/report_collector.rs

```diff
@@ -151,6 +151,11 @@ enum ReportCollectorCommand {
         /// Number of records to aggregate
         #[clap(long, short = 'n')]
         count: u32,
+
+        // If set, use the specified fixed polling interval when running a query.
+        // Otherwise, use exponential backoff.
+        #[arg(long, default_value_t = 0)]
+        set_fixed_polling_ms: u64,
```

Comment:
is `Option<u64>` better represents this being not set?

Reply:
oh, yeah, probably